onnxruntime/core/platform/windows/env.cc

```diff
@@ -113,6 +127,15 @@ class WindowsThread : public EnvThread {
   }
 #pragma warning(pop)
 
+  static void __stdcall CustomThreadMain(void* param) {
+    std::unique_ptr<Param> p((Param*)param);
+    ORT_TRY {
+      p->start_address(p->index, p->param);
+    }
+    ORT_CATCH(const std::exception&) {
```

Comment:
There is no try/catch clause in POSIX variant.

Reply:
It is because for POSIX, CustomThreadMain calls ThreadMain which already has try-catch.

Reply:
Then try/catch needs to be added WindowsThread::ThreadMain method. It is actually there, but does not cover complete function body. It's not a good idea to let exception propagate from function provided to _beginthreadex.

Actually I see no difference between CustomThreadMain and ThreadMain.

The above consideration is not critical and can be handled separately.

Reply:
Thanks, let's address that shortly then.